plugins/image-source/obs-slideshow.c

```diff
@@ -193,12 +254,28 @@ static void ss_update(void *data, obs_data_t *settings)
 	uint32_t cx = 0;
 	uint32_t cy = 0;
 	size_t count;
+	const char *behavior;
+	const char *mode;
 
 	/* ------------------------------------- */
 	/* get settings data */
 
 	da_init(new_files);
 
+	behavior = obs_data_get_string(settings, S_BEHAVIOR);
+
+	if (astrcmpi(behavior, S_BEHAVIOR_PAUSE_UNPAUSE) == 0) {
+		ss->behavior = BEHAVIOR_PAUSE_UNPAUSE;
+	} else if (astrcmpi(behavior, S_BEHAVIOR_ALWAYS_PLAY) == 0) {
+		ss->behavior = BEHAVIOR_ALWAYS_PLAY;
+	} else { /* S_BEHAVIOR_STOP_RESTART */
+		ss->behavior = BEHAVIOR_STOP_RESTART;
+	}
```

Comment:
Style:  Curly braces are not needed for this `if` statement, since the contents of each branch is a single statement.